src/lib/storage/partitioning/hash_function.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "all_type_variant.hpp"
+#include "strong_typedef.hpp"
+
+namespace opossum {
+
+class HashFunction {
```

Comment:
This just hashes an AllTypeVariant? Should be in all_type_variant.hpp then, not in a partitioning/ folder

Reply:
Also needs a virtual destructor if you actually want to inherit from this, but I don't think you do?